### PR TITLE
feat(sig): Submit proposal for a CF SIG

### PR DESCRIPTION
**Pull request by ryanpei**
Proposing a SIG to focus on the Cloud Foundry (CF) clouddriver

### CODE REVIEW

**Comment by ryanpei** [comment]
cc @zachsmith1

File: sig-cf/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# SIG: CF
+
+This SIG focuses on the current Cloud Foundry (CF) clouddriver provider for Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Last Thursday of each month, at 1 PM EST / 10AM PST
```
Comment (ezimanyi) [inline sig-cf/README.md:8]:
```suggestion
  * Last Thursday of each month, at 10 AM US/Pacific ([See in your time zone](https://www.thetimezoneconverter.com/?t=10am&tz=San%20Francisco))
```

File: sig-cf/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  [Agenda](https://docs.google.com/document/d/1TouFx5ykNCcZxNSD2JhUiRliZDXwxZ6sx239rigYdxw/edit?usp=sharing)
+
+## Leadership
+
+* Zach Smith - Armory (https://github.com/zachsmith1)
+* Ryan Pei - Armory (https://github.com/ryanpei)
+
+## Contact
+
+* [Slack](https://spinnakerteam.slack.com/archives/C4ENF2E1W)
```
Comment (ezimanyi) [inline sig-cf/README.md:19]: The convention for the other SIGs is to have the channel be called `#sig-{name}`; would it make sense to use that here to for discoverability?

Comment (ryanpei) [reply sig-cf/README.md:19]: good idea!

File: sig-cf/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Ryan Pei - Armory (https://github.com/ryanpei)
+
+## Contact
+
+* [Slack](https://spinnakerteam.slack.com/archives/C012PGYN771)
+
+## Scope
+
+*Status and Alignment*
+  * [CF Clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-cloudfoundry)
```
Comment (ajordens) [inline sig-cf/README.md:24]: This appears to be narrowly scoped to just cloudfoundry in `clouddriver` ... I assume this SIG will also cover CF-related issues that may pop up in the UI and orchestration tiers?

Comment (ryanpei) [reply sig-cf/README.md:24]: Yes, is this more clear?
```suggestion
  * Issues in [CF Clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-cloudfoundry) and other Spinnaker services which affect deployments to CloudFoundry
```

File: sig-cf/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    * Review issues and PRs
```
Comment (ryanpei) [inline sig-cf/README.md:25]: remove this line if the suggestion above is preferred
```suggestion
```

**Comment by zachsmith1** [comment]
LGTM!

**Pull request merged**